docs/build_node.md

```diff
@@ -378,14 +378,22 @@ while (true) {
 
 **Dependencies:** `ChannelManager`, `ChainMonitor`
 
-### Once Per Minute: `ChannelManager`'s `timer_chan_freshness_every_min()`
-**What it's used for:** `ChannelManager` needs to be told every time a minute passes so that it can broadcast fresh channel updates if needed
+### Background Processing
+
+`ChannelManager`'s `timer_chan_freshness_every_min()`
```

Comment:
Could we remove this line? I'd like to keep the format of the steps consistent, so 1) name of step, 2) what it's used for, 3) example, etc, and this line breaks that.

Reply:
Whoops, meant to delete that.